arelle/plugin/validate/NL/rules/nl_kvk.py

```diff
@@ -92,3 +92,25 @@ def rule_nl_kvk_3_1_1_2(
                 modelObject = val.modelXbrl
             )
             return
+
+
+@validation(
+    hook=ValidationHook.XBRL_FINALLY,
+)
+def rule_nl_kvk_3_1_3_1 (
+        pluginData: PluginValidationDataExtension,
+        val: ValidateXbrl,
+        *args: Any,
+        **kwargs: Any,
+) -> Iterable[Validation]:
+    """
+    NL-KVK.3.1.3.1: xbrli:segment must not be used in contexts.
+    """
+    contexts = val.modelXbrl.contexts.values()
+    for context in contexts:
+        if context.hasSegment:
+            yield Validation.error(
```

Comment:
Perhaps it would be better to raise a single error with references to all offending contexts? If the report was created using segments, it's likely that all of the contexts in the report were.